src/status_im/ui/screens/bottom_sheets/views.cljs

```diff
@@ -47,10 +48,16 @@
           (merge key-storage/migrate-account-password)
 
           (= view :pinned-messages-list)
-          (merge {:content pin.list/pinned-messages-list}))]
+          (merge {:content pin.list/pinned-messages-list})
+          
+          (= view :drawer/reactions)
+          (merge {:content drawers/reactions}))]
     (reagent/create-class {:reagent-render         (fn []
```

Comment:
No suggestion for this PR, just sharing that we could use `https://reagent-project.github.io/docs/master/reagent.core.html#var-with-let` for the case of rendering a component once and doing something when it's unmounted.

Another future proof approach is to embrace React hooks and stop using the imperative style.

Reply:
It's more like a general feedback for our entire codebase. In theory, we should move away from the classical React component lifecycle methods, as this is what most React codebases have been doing in the last couple years. So yes, this means using React hooks instead.